src/lib/ngsiNotify/Notifier.cpp

```diff
@@ -92,50 +92,57 @@ static bool templateNotify
     verb = POST;
   }
   method = verbName(verb);
-
+  LM_W(("KZ: In templateNotify: method: '%s'", method.c_str()));
 
   //
   // 2. URL
   //
   macroSubstitute(&url, httpInfo.url, ce);
-
+  LM_W(("KZ: url: '%s'", url.c_str()));
 
   //
   // 3. Payload
   //
   if (httpInfo.payload == "")
   {
+    LM_W(("KZ: using default payload"));
     payload = "{" + ce.toJson(renderFormat, attrsOrder) + "}";
   }
   else
   {
     macroSubstitute(&payload, httpInfo.payload, ce);
+    renderFormat = NGSI_V2_CUSTOM;
```

Comment:
Here is the fix for "custom" as http header 'attrValues' when payload isn't the default payload